### ISSUE TITLE
Breakline does not work

### ISSUE THREAD

**Issue by iranadryan** [gitbrent/PptxGenJS#730]
`array.push({ text: m[1], options: { color: '0070C0', breakLine: false } });`

**Comment by iranadryan** [comment]
I put as false and it contiues putting a \n

**Comment by ankon** [comment]
Which version of PptxGenJS are you using?

**Comment by Offbeatmammal** [comment]
I am using the CDN version https://cdn.jsdelivr.net/gh/gitbrent/pptxgenjs@latest/dist/pptxgen.min.js

I have a similar issue ... not specifying blankLine is putting a single linebreak between adjacent items in the array, putting blankLine either as true or false adds an extra one

**Comment by bonavida** [comment]
This is not working either when you use [word-level formatting inside table cells](https://github.com/gitbrent/PptxGenJS/blob/ff51e874cbab989b54375021a90dd0c5e63d36d9/README.md#table-cell-formatting-examples).

```js
// Example
var arrObjText = [
    { text: 'Red ', options: { color: 'FF0000', breakLine: false } },
    { text: 'Green ', options: { color: '00FF00', breakLine: true } },
    { text: 'Blue', options: { color: '0000FF' } }
];
slide.addText(arrObjText, {});
```
outputs
```
Red
Green
Blue
```
instead of
```
Red Green
Blue
```

**Comment by gitbrent** [comment]
While there continues to be issues under development with using `align` sometimes, this particular example works as designed (v3.2.0):

```javascript
var pptx = new PptxGenJS();
var slide = pptx.addSlide();

slide.addText(
    [
        { text: "Red ", options: { color: "FF0000", breakLine: false } },
        { text: "Green ", options: { color: "00FF00", breakLine: true } },
        { text: "Blue", options: { color: "0000FF" } },
    ],
    { x: 1, y: 1, w: "80%", h: 3, fill: pptx.SchemeColor.background2 }
);

pptx.writeFile("PptxGenJS-Issue-730");
```
<img width="479" alt="Screen Shot 2020-05-23 at 23 04 55" src="https://user-images.githubusercontent.com/7218970/82745428-d8946a00-9d49-11ea-9152-1b53ac40184f.png">

**Comment by Offbeatmammal** [comment]
hopefully the align: issues can be addressed at some point ... I have a specific example where based on a template we have a textbox which is align:center to position titles appropriately, but if there is formatting in the line (eg one word bolded) then this issue causes extraneous linebreaks to appear.

**Comment by iranadryan** [comment]
> Which version of PptxGenJS are you using?

^3.1.1

**Comment by gitbrent** [comment]
FYI: Fixed via Issue #751
